buildpacks/nodejs-npm-engine/CHANGELOG.md

```diff
@@ -7,6 +7,50 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+- Added npm version 9.2.0.
```

Comment:
Should the automation sort the added entries by version?

Reply:
yeah, it should.  i fixed a bug in the automation where we were only ever capturing the first entry when more than one mirror distribution is detected but didn't note the order is random in the full list.

Reply:
@edmorley this should be fixed with #621.  i'm going to close this PR and wait for the automation to run again with the sorted list.